packages/dashboard/lib/s3.ts

```diff
@@ -7,11 +7,13 @@ declare const global: typeof globalThis & {
 const s3 =
   global.s3 ||
   new S3Client({
+    endpoint: process.env.S3_ENDPOINT,
     region: process.env.S3_REGION,
     credentials: {
       accessKeyId: process.env.S3_ACCESS_KEY_ID ?? '',
       secretAccessKey: process.env.S3_SECRET_ACCESS_KEY ?? '',
     },
+    forcePathStyle: !!process.env.S3_ENDPOINT,
```

Comment:
I'm not sure to understand what this option does, could you explain?

Reply:
Normally S3 uses subdomains somehow like `bucket.region.s3provider.com`.  By setting forcePathStyle S3 is using urls like `s3provider.com/region/bucket` as minio is only listening on localhost:someport it can not handle subdomains of localhost, but needs the path style.